package.json

```diff
@@ -17,12 +17,11 @@
     "test": "npm run static-analysis && npm run jest",
     "test:clean": "npm run clean && npm run test",
     "test:automation": "wdio wdio.conf.js",
-    "test:automation:oneOnOne": "wdio wdio.conf.js --suite oneOnOne",
+    "test:automation:smoke": "wdio wdio.conf.js --suite smoke",
     "test:automation:space": "wdio wdio.conf.js --suite space",
     "test:automation:recents": "wdio wdio.conf.js --suite recents",
-    "test:automation:multiple": "wdio wdio.conf.js --suite multiple",
     "test:tap": "cross-env TAP=true wdio wdio.conf.js --suite tap",
-    "test:integration": "cross-env INTEGRATION=true wdio wdio.conf.js --suite integration",
+    "test:integration": "cross-env INTEGRATION=true wdio wdio.conf.js --suite smoke",
```

Comment:
Keeping the "integration" name and just pointing it to smoke. That way we don't have to modify circle or jenkins.